galaxy_ng/app/tasks/index_registry.py

```diff
@@ -0,0 +1,160 @@
+import json
+import logging
+from urllib.parse import quote, urlencode
+
+from django.core import exceptions
+from django.utils.translation import gettext_lazy as _
+from django.http.request import HttpRequest
+
+from pulpcore.plugin.tasking import dispatch
+
+from pulp_container.app import models as container_models
+
+from galaxy_ng.app.api.ui import serializers
+from galaxy_ng.app import models
+
+
+log = logging.getLogger(__name__)
+
+
+CATALOG_API = "https://catalog.redhat.com/api/containers/v1/repositories"
```

Comment:
should this go to settings?
```suggestion
CATALOG_API = settings.get("CATALOG_API",  "https://catalog.redhat.com/api/containers/v1/repositories")
```

Reply:
I'm torn on this. I think I'm going to keep it out of settings for now. I can't think of a good reason why anyone would need to edit this value